src/psyclone/tests/psyGen_test.py

```diff
@@ -2569,6 +2579,7 @@ def test_loop_props():
 def test_node_abstract_methods():
     ''' Tests that the abstract methods of the Node class raise appropriate
     errors. '''
+    from psyclone.psyGen import Node
```

Comment:
pylint complains that `Node` has already been imported.

Reply:
Fixed.